grammar.js

```diff
@@ -82,8 +82,6 @@ module.exports = grammar({
     /[ \t\f\r\n;]/
   ],
 
-  word: $ => $.identifier,
-
```

Comment:
Unrelated: This rule isn't used anywhere. Not sure why this exists, but, no point keeping it.

Reply:
Ah, never mind! "word" is a special rule that helps tree-sitter recognize keywords: https://tree-sitter.github.io/tree-sitter/creating-parsers#keyword-extraction